hail/python/hail/expr/functions.py

```diff
@@ -266,6 +266,16 @@ def typecheck_expr(t, x):
                             .format(dtype)) from e
 
     if wrapper['has_expr']:
+        if ( x._ir.free_vars.__len__() > 0 or
+             x._ir.free_agg_vars.__len__() > 0 or
+             x._ir.free_scan_vars.__len__() > 0
```

Comment:
Better form to use `len(...)`

Reply:
Can use len as it's aliased in the same file :(

Reply:
I could use `buildtins.len`, but not sure what advantage that has